script/M2_Deploy.s.sol

```diff
@@ -34,12 +38,13 @@ contract Deployer_M2 is ExistingDeploymentParser {
         uint256 configChainId = stdJson.readUint(config_data, ".chainInfo.chainId");
         require(configChainId == currentChainId, "You are on the wrong chain for this config");
 
-        address oracleInitialOwner = executorMultisig;
-        uint256 initialThreshold = stdJson.readUint(config_data, ".oracleInitialization.threshold");
+        oracleInitialOwner = executorMultisig;
+        initialThreshold = stdJson.readUint(config_data, ".oracleInitialization.threshold");
         bytes memory oracleSignerListRaw = stdJson.parseRaw(config_data, ".oracleInitialization.signers");
-        address[] memory initialOracleSigners = abi.decode(oracleSignerListRaw, (address[]));
+        initialOracleSigners = abi.decode(oracleSignerListRaw, (address[]));
 
         require(initialThreshold <= initialOracleSigners.length, "invalid initialThreshold");
+        require(initialThreshold >= 1, "invalid initialThreshold");
```

Comment:
We don't need the beacon chain oracle file actually, this is deployed by Succinct.

We do need all of the contracts in this script: https://github.com/Layr-Labs/eigenlayer-contracts/blob/m2-upgrade-script/script/upgrade/GoerliM2Upgrade.s.sol

Could you add them?

Reply:
🫡 added